src/js/components/app/App.js

```diff
@@ -37,6 +50,16 @@ function App(props) {
     window.onbeforeunload = () => dirty ? true : undefined
   }, [dirty])
 
+  useEffect(() => {
+    if (!askingToRestore) return
+    const msg = 'You have some unsubmitted data for this form. Would you like to pick up where you left off?'
+    if (confirm(msg)) {
+      acceptLocalStorage()
+    } else {
+      rejectLocalStorage()
+    }
+  }, [askingToRestore])
```

Comment:
In the root component we pull `askingToRestore` out of the UI section of our store's state; that's a flag that gets set when we find unsubmitted data in local storage that the user might want to continue with. When the flag is true we prompt the user and ask if we should _accept_ or _reject_ the stored data. We then dispatch an action in repose (`ACCEPT_LOCAL_STORAGE` or `REJECT_LOCAL_STORAGE`, accordingly).